extensions/publisher-cves/vite.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from "vite";
+import vue from "@vitejs/plugin-vue";
+import tailwindcss from "@tailwindcss/vite";
+
+// https://vite.dev/config/
+export default defineConfig({
+  base: "",
+  plugins: [vue(), tailwindcss()],
+  server: {
+    proxy: {
+      "/api": {
+        target: "http://localhost:8000",
+        changeOrigin: true,
```

Comment:
Mostly setup from Vite's templating other than this which is to develop locally alongside the FastAPI server.